app/views/govuk_publishing_components/components/_government_navigation.html.erb

```diff
@@ -5,37 +5,37 @@
 <nav id="proposition-menu" class="no-proposition-name gem-c-government-navigation" aria-label="Departments and policy navigation">
   <ul id="proposition-links">
     <li>
-      <a class="<%= 'active' if active == 'departments' %>" href="/government/organisations">
+      <a class="<%= 'active' if active == 'departments' %> govuk-link govuk-link--no-underline govuk-link--inverse" href="/government/organisations">
```

Comment:
This is off both on the change (left) and live (right)

![image](https://user-images.githubusercontent.com/71266765/120331804-11183d80-c2e6-11eb-8290-44d42bd263da.png)

Reply:
This is outlined in https://github.com/alphagov/govuk_publishing_components/pull/2081 In brief: this is intentional and noted in the docs.

Reply:
Ahh - I see this now, this is confusing on the gem docs.

 🤔  I don't think these "example" pulls-outs are a good use in this context. They're not reflective on the rendered element, I'm wondering wondering if 
 
- a supplementary image, 
- changing the "example background" to black, 
- changing the copy to a [warning](https://components.publishing.service.gov.uk/component-guide/warning_text) element might be clearer.

However equally there's note is there so not a blocker.